common/source/docs/common-escs-and-motors.rst

```diff
@@ -29,19 +29,45 @@ Motors
 [/site]
 
 
-ESC 
-====
+ESC for Brushless Motors 
+========================
+
+ESCs (Electronic Speed Controls) have many different protocols for communicating with the autopilot for motor speed control, and and can also provide telemetry information on motor rpm, battery voltage. The
+capabilities will vary with each individual ESC model. Some ESCs have specialized firmware that allows easy configuration of operating parameters. The following sections
+explain the required ArduPilot setup to utilize the protocols, telemetry, and setup programs that various ESCs utilize.
+See :ref:`common-esc-guide` for a guide to terminology.
+
+Protocols
+---------
+
 .. toctree::
     :maxdepth: 1
 
-    BLHeli/BLHeli32 Capable ESCs <common-blheli32-passthru>
-    Currawong Velocity ESCs <common-velocity-can-escs>
-    DroneCAN ESCs <common-uavcan-escs>
+    PWM, OneShot and OneShot125 ESCs <common-brushless-escs>
     DShot ESCs <common-dshot-escs>
+    DroneCAN ESCs <common-uavcan-escs>
```

Comment:
Previously it was in alphabetical order.. I guess it is in order of popularity now.. I don't mind but just pointing this out.

Reply:
Historical, but I should move KDE can up above Currawong